dap_prinz_green_jobs/getters/occupation_getters.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+
+
+def load_job_title_soc() -> pd.DataFrame():
+    """
+    Load the dataset which gives SOC codes for thousands of job titles
+    """
+
+    jobtitle_soc_data = pd.read_excel(
+        "s3://prinz-green-jobs/inputs/data/occupation_data/ons/indexsocextv5updated.xlsx",
+        sheet_name="SOC 2020 6 Digit Index",
+        converters={"SOC 2020 Ext Code": str, "SOC 2020": str, "SOC 2010": str},
+    ).rename(
+        columns={
+            "SOC 2020 Ext Code": "SOC_2020_EXT",
+            "SOC 2020": "SOC_2020",
+            "SOC 2010": "SOC_2010",
+        }
+    )
+
+    return jobtitle_soc_data
+
+
+def load_green_gla_soc() -> pd.DataFrame():
+    """
+    Load the greater london authority dataset which gives green measures from SOC codes
+    """
+
+    green_gla_data = (
+        pd.read_excel(
+            "s3://prinz-green-jobs/inputs/data/occupation_data/gla/Summary of green occupations (Nov 2021).xlsx",
+            sheet_name="1. List of all occupations",
+            skiprows=3,
+            converters={"SOC2010 4-digit": str},
+        )
+        .add_prefix("GLA_")
```

Comment:
I feel like the getter should just load data not load and modify over and over - I'd pull all these cleaning steps across all the load_ getters into an` occupations_data_processing.py` script that outputs clean datasets + you can always add additional cleaning steps for other datasets there and then i.e.:

```suggestion
   return load_s3_data(BUCKET_NAME, 'outputs/data/green_occupations/clean_green_gla_soc.csv)
```

Reply:
liz - to do